api.go

```diff
@@ -20,11 +20,12 @@ import (
 )
 
 type Client struct {
-	Token         string
-	SIOEndpoint   url.URL
-	Http          http.Client
-	Insecure      string
-	ShowBody      bool
+	Token       string
+	SIOEndpoint url.URL
+	Http        http.Client
+	Insecure    string
+	ShowBody    bool
+
 	configConnect *ConfigConnect
 }
 
```

Comment:
This looks like just an issue of Whitespace, can we fix this so we don't merge a whitespace diff?